android/expoview/src/main/java/host/exp/exponent/experience/ErrorConsoleFragment.kt

```diff
@@ -0,0 +1,65 @@
+package host.exp.exponent.experience
+
+import android.widget.ArrayAdapter
+import android.app.Activity
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import host.exp.exponent.Constants
+import host.exp.exponent.kernel.ExponentError
+import host.exp.expoview.databinding.ErrorConsoleFragmentBinding
+
+class ErrorConsoleFragment : Fragment() {
+  private var _binding: ErrorConsoleFragmentBinding? = null
+  private val binding get() = _binding!!
```

Comment:
More of a question than a suggestion -- why use _binding instead of a lateinit private var? Is it so that the member variable is readonly?

Reply:
This was how the official docs suggested it be set up for fragments: https://developer.android.com/topic/libraries/view-binding#fragments

I think the reason is that setting a lateinit to null isn't supported, and binding needs to be set to null in `onDestroy` since technically the binding is no longer valid.

Reply:
Thanks for the link and pointing out the cleanup!